project/AppDependencies.scala

```diff
@@ -8,7 +8,8 @@ object AppDependencies {
   val compile = Seq(
     "uk.gov.hmrc" %% "play-reactivemongo" % "6.2.0",
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-25" % "3.3.0"
+    "uk.gov.hmrc" %% "bootstrap-play-25" % "3.3.0",
+    "com.github.java-json-tools" % "json-schema-validator" % "2.2.8"
```

Comment:
Do we still use this dependency?

Reply:
@adamconder  This is form schema validation.